export.py

```diff
@@ -2061,7 +2062,7 @@ def export_object_attributes(ri, scene, ob, visible_objects):
     obj_groups_str = '*'
     for obj_group in scene.renderman.object_groups:
         if ob.name in obj_group.members.keys():
-            obj_groups_str += ',' + obj_group.name
+            obj_groups_str = obj_group.name
```

Comment:
So this change looks good.  I think this might be the only line I'd disagree with.  The problem is right now you can't support being part of two obj_groups.  Is this intended?